features/actions.feature

```diff
@@ -10,26 +10,41 @@ Feature:  Openhab Action Support
       rule 'Notify a user' do
         on_start
         run do
-          user = 'user@openhab.com'
-          notify(user: user, msg: 'Test Alert')
+          user = 'user@example.com'
+          notify('Test Alert', email: user)
           logger.info("Notified #{user}")
          end
       end
       """
     When I deploy the rule
-    Then It should log 'Notified user@openhab.com' within 5 seconds
+    Then It should log 'Notified user@example.com' within 5 seconds
 
   @not_implemented
   Scenario: send to all users
     Given a rule
       """
-      rule 'Notify a user' do
+      rule 'Notify all user' do
         on_start
         run do
-          notify_all('Test Alert')
+          notify('Test Alert')
           logger.info("Notified All Users")
          end
       end
       """
     When I deploy the rule
-    Then It should log 'Notified All users' within 5 seconds
\ No newline at end of file
+    Then It should log 'Notified All users' within 5 seconds
+
+  Scenario: Execute a command line
+    Given a rule
+      """
+      java_import java.time.Duration
+      rule 'Execute command line' do
+        on_start
+        run do
+          output = Exec.executeCommandLine(Duration.ofSeconds(2), '/bin/echo', 'Hello, World!').chomp
```

Comment:
Should we support '2.seconds' here instead?  
WDYT?

Reply:
This is just how the raw Exec action works. We can provide a special method e.g. exec_command_line which we can support `2.seconds`. I am not yet familiar with the duration handling.

Reply:
I am not sure we need to... I need to experiment, It is possible if in the Ruby duration class we create a method called "to_java" and have it return the Java Duration here, Jruby it might just automatically convert it for us.

Reply:
Do you want me to check this?

Reply:
> Do you want me to check this?

yes please

Reply:
Ok, talked with the JRuby group and its not possible.  If we want this it would be better to return Java.Time.Duration rather than the duration object I built.  I will open an story to look at that. But it shouldn't hold this up.